posts/2020/03/documentation-as-a-way-to-build-community.md

```diff
@@ -0,0 +1,60 @@
+<!--
+.. title: Documentation as a way to build Community
+.. slug: documentation-as-a-way-to-build-community
+.. date: 2020-03-10 15:25:55 UTC-05:00
+.. author: Melissa Mendonça
+.. tags: Labs, NumPy
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+As a long time user and participant in open source communities, I've always known that documentation is far from being a solved problem. At least, that's the impression we get from many developers: "writing docs is boring"; "it's a chore, nobody likes to do it". I have come to realize I'm one of those rare people who likes to write both code and documentation. 
+
+Nobody will argue against documentation. It is clear that for an open-source software project, documentation is the public face of the project. The docs influence how people interact with the software and with the community. It sets the tone about inclusiveness, how people communicate and what users and contributors are able to do. Looking at the results of a “NumPy Tutorial” search on any search engine also gives an idea of the demand for this kind of content - it is possible to find documentation about how to read the NumPy documentation!
+
+I've started working at Quansight in January, and I have started doing work related to the [NumPy CZI Grant](https://labs.quansight.org/blog/2019/11/numpy-openblas-CZI-grant/). As a former professor in mathematics, this seemed like an interesting project both because of its potential impact on the NumPy (and larger) community and because of its relevance to me, as I love writing educational material and documentation. Having official high-level documentation written using up-to-date content and techniques will certainly mean more users (and developers/contributors) are involved in the NumPy community.
+
+So, if everybody agrees on its importance, why is it so hard to write good documentation?
+
+## Why do we lack documentation?
+
+In a recent article about documentation for open source data analytics libraries, [Geiger et al.](https://doi.org/10.1007/s10606-018-9333-1) points out that "In a [2017 GitHub survey](https://opensourcesurvey.org/2017/) of OSS contributors, 93% reported that “incomplete or outdated documentation is a pervasive problem” but “60% of contributors say they rarely or never contribute to documentation” (Zlotnick et al. 2017)." At the same time, still in the words of Geiger et al., "Many interviewees who regularly contribute documentation to such projects stated that they did not feel like they received same levels of positive community feedback for documentation work as they did for adding new features or fixing bugs." In fact, the authors in this paper go as far as saying that documentation is the *invisible work* behind these projects. 
```

Comment:
minor: "points out" -> "point out"

Reply:
The two levels of quotation marks is a little confusing. How about using italics for the inner level to help with that?